src/components/MapControls/MapControls.js

```diff
@@ -40,13 +43,102 @@ const degreesToRadians = (degrees) => {
   return degrees * (Math.PI / 180);
 };
 
+const useStyles = makeStyles((theme) => {
+  return {
+    mapControls: {
+      position: "absolute",
+      display: "flex",
+      flexDirection: "column",
+      gap: 8,
+      right: (props) => (props.isMobile ? 15 : props.overlayWidth + 15),
```

Comment:
it should be 10 instead of 15 on small width screen

Reply:
OK